components/prism-cmake.js

```diff
@@ -1,8 +1,18 @@
-Prism.languages.cmake = Prism.languages.extend('python', {
+Prism.languages.cmake = {
+    'comment': {
+		pattern: /(^|[^\\])#.*/,
```

Comment:
Now that we control the comment pattern, we can simplify it!
This is simpler and should also work:
```js
'comment': /#.*/
```